indian-journal-of-medical-research.csl

```diff
@@ -237,8 +213,8 @@
     <sort>
       <key variable="citation-number"/>
     </sort>
-    <layout prefix="(" suffix=")" delimiter=",">
-      <text variable="citation-number"/>
+    <layout vertical-align="sup" delimiter=",">
```

Comment:
what's going on here? Are these superscript or not? Presumably yes, right? If so, pls. remove the `vertical-align="baseline"` below - thanks!

Reply:
Looks like this should be superscript, yes (checked in https://doi.org/10.4103/ijmr.IJMR_616_18). I'll go ahead and make the change and merge.